test/node/native/native.node.test.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @vitest-environment jsdom
```

Comment:
there's vitest env react-native but i'm not sure whether it strips unsupported global node modules.

Reply:
I'll give that a shot - must have missed that in vitest docs!